### ISSUE TITLE
请问在使用提供的预训练权重，去除位置信息后，训练结果不如论文提供的数据。

### ISSUE THREAD

**Issue by Zoeun** [shannanyinxiang/SPTS#6]
修改了pt_label = torch.cat([center_pts, recog_label], dim=-1）为pt_label = recog_label，评估指标编辑距离作为匹配规则。使用提供的预训练权重，训练icdar2013数据集，召回率低于40%，f分数在60左右，请问您的训练过程，如何进行修改？

**Comment by shannanyinxiang** [comment]
在Issue #1 中提供的预训练权重是使用了单点坐标训练的。训练无点模型需要重新进行预训练。

**Comment by Zoeun** [comment]
请问能否提供无位置坐标的预训练模型

**Comment by shannanyinxiang** [comment]
不好意思，我现在自己也没有这个预训练模型。无点的预训练模型在之前实习的公司里。

**Comment by yangxcccscsa** [comment]
> 修改了pt_label = torch.cat([center_pts, recog_label], dim=-1）为pt_label = recog_label，评估指标编辑距离作为匹配规则。使用提供的预训练权重，训练icdar2013数据集，召回率低于40%，f分数在60左右，请问您的训练过程，如何进行修改？

您好，我是一名做文字检测识别的研究生，我可以加您微信问您一些问题吗？我的微信号：ZY18220759108
